lib/connect-session-sequelize.js

```diff
@@ -33,6 +33,10 @@ module.exports = function SequelizeSessionInit (Store) {
         throw new SequelizeStoreException('Database connection is required')
       }
 
+      if (options.secret) {
+        options.kruptein = require('kruptein')(options)
```

Comment:
You pass the full `options` to `kruptein` which contains all options given to `connect-session-sequelize`. However [none of those options are relevant for `kruptein`](https://github.com/jas-/kruptein#options).

Reply:
Correct, while the entire options object is used only those applicable are applied within the construct. This is intentional to limit api changes. Thanks